resolver-dns/src/main/java/io/netty/resolver/dns/ResolvConf.java

```diff
@@ -81,6 +81,13 @@ private ResolvConf(BufferedReader reader) throws IOException {
 
             if (ln.startsWith("nameserver")) {
                 ln = ln.substring("nameserver".length()).trim();
+                int cIndex = ln.indexOf('#');
+                if (cIndex != -1) {
+                    ln = ln.substring(0, cIndex).trim();
+                }
```

Comment:
Feel free to ignore: 
```suggestion
                ln = ln.substring("nameserver".length());
                int cIndex = ln.indexOf('#');
                if (cIndex != -1) {
                    ln = ln.substring(0, cIndex);
                }
                ln = ln.trim();
```